tmt.spec

```diff
@@ -16,6 +16,7 @@ URL: https://github.com/teemtee/tmt
 Source0: https://github.com/teemtee/tmt/releases/download/%{version}/tmt-%{version}.tar.gz
 
 %define workdir_root /var/tmp/tmt
+%define usr_local_bin /usr/local/bin
```

Comment:
Is there a reason to use `/usr/local/bin`? These scripts will come from the rpm, so `/usr/bin` is the correct location?

https://www.pathname.com/fhs/pub/fhs-2.3.html#PURPOSE23